src/zoid/buttons/util.js

```diff
@@ -214,8 +219,7 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
                     session.completePayment(newUpdate);
                 },
                 begin: () => session.begin(),
-                abort: () => session.abort(),
-                oncancel: () => session.oncancel()
```

Comment:
@andyfen Why are we refactoring this?  I think the issue is that when I look at SPB, the `oncancel` callback isn't being brought in from the response: https://github.com/paypal/paypal-smart-payment-buttons/blob/c12f7f31540246d593e53e3bbd4e68d29218817b/src/payment-flows/applepay/applepay.js#L345

Reply:
I see what happened.  This wasn't here before and was added.  Interesting this doesn't work though.  This should be fine.  Did we update the Apple Pay version.  I don't recall `abort` being supported last year.